test/handler.spec.ts

```diff
@@ -1,20 +1,7 @@
-// import * as mocha from "mocha";
-import * as chai from "chai";
+import test from "ava";
```

Comment:
Is there a reason we're using ava and not a regular jest test? We use jest internally so it makes sense to use it here too.

Reply:
Well, I have changed the testing framework back to jest :)